kasenvr/project-athena
# Whitelist disabled by default, now has a toggle

The description was also further revised.

**To test:**
When you first launch interface, ensure the whitelist is off by default, you will know by opening the settings menu and seeing that the toggle is unchecked. Enable the whitelist on the toggle and then reload scripts, look at the log to see that the whitelist is active. When the whitelist is inactive, it should output for every script that it passes the whitelist status that it is in fact: disabled.

Close interface with the whitelist toggled to "on", reopen interface and ensure it is on again. Do the same for the "off" state to ensure it is performing as expected when set to on/off.
